util/stmtsummary/statement_summary.go

```diff
@@ -72,6 +72,19 @@ type stmtSummaryByDigestMap struct {
 
 	// sysVars encapsulates system variables needed to control statement summary.
 	sysVars *systemVars
+
+	evictedInfo stmtSummaryEvictedInfo
+	evictedInfoHistory
+	evictedInfoCount int
```

Comment:
how about putting this into `evictedInfoHistory`?

Reply:
Ofc😄